src/containers/hashlog/Hashlog.js

```diff
@@ -0,0 +1,161 @@
+
+const {Screen} = createMqProvider(querySchema);
+
+class Hashlog extends React.Component {
+  state = {
+    selectedRound: 0,
+    nameFilter: '',
+    sort: {},
+  };
+
+  componentDidMount() {
+    const {getBlocksPages, hashlog : {blocksPage, blocksPageSize}} = this.props;
+    getBlocksPages(blocksPage, blocksPageSize);
+  }
+
+  onRowClick = rowData=> {
+    const { setActionBlock,  history} = this.props;
+    history.push('/action-list?number=' + rowData.original.number);
+    setActionBlock(rowData.original);
+  }
+
+  renderBlocklist = () => {
+    const {
+      hashlog: {blockList, totalBlocks, blocksPage, blocksPageSize},
+      setBlocksPage,
+      setBlocksPageSize,
+      getBlocksPages
+    } = this.props;
+    return (
+      <React.Fragment>
+        <Screen on={screenWidth => (
+          <ReactTable
+            data={blockList}
+            columns={this.getColumns(screenWidth)}
+            pages={Math.ceil(totalBlocks/blocksPageSize)}
+            page={blocksPage}
+            defaultPageSize={blocksPageSize}
+            pageSize={blocksPageSize}
+            showPagination={true}
+            screenWidth={screenWidth}
+            showPaginationBottom={true}
+            manual
+            paginationPageDispatcher={page => {
+              setBlocksPage(page);
+              getBlocksPages(page, blocksPageSize);
+            }}
+            paginationPageSizeDispatcher={pageSize => {
+              setBlocksPageSize(pageSize);
+              getBlocksPages(blocksPage, pageSize);
+            }}
+            onItemSelected={() => {}}
+            PaginationComponent={PaginationWithPage}
+          />)}
+        />
+      </React.Fragment>
+    );
+  }
+
+  getColumns = screenWidth => {
+    return [
+      {
+        Header: this.props.intl.messages['hashlog.blockNumber'],
+        id: 'currency',
+        accessor: 'number',
+        headerClassName: 'hashlog__table-header-title',
+        className: 'table_col_value hashlog__table-cell hashlog__table-cell-number-wrapper',
+        minWidth: screenWidth === 'lg' ? 60 : 25  ,
+        Cell: row => (
+          <a
+            className="hashlog__table-cell-number"
+            href={'/action-list?number=' + row.original.number}
+            onClick={ e => {
+              e.preventDefault();
+              this.onRowClick(row);
+            }}>
+            {row.original.number}
+          </a>
+        ),
+      },
+      {
+        Header: this.props.intl.messages['hashlog.blockHash'],
+        className: 'table_col_value hashlog__table-cell hashlog__table-cell_hash-value',
+        headerClassName: 'hashlog__table-header-title',
+        minWidth: 100,
+        accessor: 'hash'
+      },
+      {
+        Header: this.props.intl.messages['hashlog.actionCount'],
+        className: 'table_col_value hashlog__table-cell',
+        headerClassName: 'hashlog__table-header-title',
+        minWidth:  screenWidth  === 'lg' ? 100 : 30,
```

Comment:
Width shouldn't be so huge. The column contains several numbers, bu it has the same width as hash column with 64 digit content.

The same for block number column.